plugins/com.google.cloud.tools.eclipse.preferences/src/com/google/cloud/tools/eclipse/preferences/messages.properties

```diff
@@ -0,0 +1,3 @@
+FIELD_EDITOR_ANALYTICS_DISCLAIMER=Your use of this plugin is subject to the Apache 2.0 software license and the <a href="privacy-policy">Google Privacy Policy</a>.
```

Comment:
I think it's worth putting the link in here too.  Since this would be translated, the href should also point to the appropriate language text page